src/pages/[platform]/build-a-backend/auth/index.mdx

```diff
@@ -13,18 +16,26 @@ export const meta = {
     'react-native',
     'swift',
     'vue'
-  ]
+    ],
+    route: '/[platform]/build-a-backend/auth'
 };
 
 export const getStaticPaths = async () => {
-  return getCustomStaticPath(meta.platforms);
+    return getCustomStaticPath(meta.platforms);
 };
 
 export function getStaticProps(context) {
-  return {
+    const childPageNodes = getChildPageNodes(meta.route);
+    return {
     props: {
-      platform: context.params.platform,
-      meta
+        platform: context.params.platform,
+        meta,
+        childPageNodes
     }
-  };
+    };
 }
+
+# Auth
```

Comment:
Can we remove these h1 from new pages we add? Since the existing page content we migrated in doesn't have any h1, I added some logic [to programmatically output the h1](https://github.com/aws-amplify/docs/commit/be6490525bebe779493669097b54feda471c978d)

Reply:
Those should all be gone now, staging site is rebuilding.